### PR TITLE
Implement Tracker.Computation.prototype.onStop

### LINKED ISSUE #4: Implement Tracker.Computation.prototype.onStop
New version of Tracker has now `onStop` method. We should provide that on the server side as well.

### PR DESCRIPTION
**Pull request by cherbst**
fixes #4

### CODE REVIEW

File: server.coffee [hunk 1]
```diff
@@ -189,11 +190,27 @@ class Tracker.Computation
...
+        f @
+    else
+      @_onStopCallbacks.push f
+
   stop: ->
     if not @stopped
       @stopped = true
       @invalidate()
 
+      @stopped = true
```
Comment (mitar) [inline server.coffee]: This is needed?

**Comment by mitar** [comment]
Thanks!

File: server.coffee (hunk 1 above)
Comment (cherbst) [inline server.coffee]: Probably not. I will change the PR.

**Comment by mitar** [comment]
Oh, you should write a comment so that I could see that you updated the pull request. I do not get a notification otherwise.

**Pull request merged**